bootcompiler/src/main/scala/org/mozartoz/bootcompiler/Main.scala

```diff
@@ -38,31 +40,38 @@ case class Config(
 /** Entry point for the Mozart2 bootstrap compiler */
 object Main {
   /** Executes the Mozart2 bootstrap compiler */
-  def main(args: Array[String]) {
+  def main(args: Array[String]): Unit = {
     // Define command-line options
     val optParser = new scopt.OptionParser[Config]("bootcompiler") {
       head("bootcompiler", "2.0.x")
-      opt[Unit]("baseenv") action {
-        (_, c) => c.copy(mode = Config.Mode.BaseEnv)
-      } text("switch to base environment mode")
-      opt[String]('o', "output") action {
-        (v, c) => c.copy(outputStream =
-          () => new BufferedOutputStream(new FileOutputStream(v)))
-      } text("output file")
-      opt[String]('m', "module") action {
-        (v, c) => c.copy(moduleDefs = v :: c.moduleDefs)
-      } text("module definition file or directory")
-      opt[String]('b', "base") action {
-        (v, c) => c.copy(baseDeclsFileName = v)
-      } text("path to the base declarations file")
-      opt[String]('D', "define") action {
-        (v, c) => c.copy(defines = c.defines + v)
-      } text("add a symbol to the conditional defines")
-      arg[String]("<file>") action {
-        (v, c) => c.copy(fileName = v)
-      } text("input file")
+
+      opt[Unit]("baseenv")
+        .action((_, c) => c.copy(mode = Config.Mode.BaseEnv))
+        .text("switch to base environment mode")
+
+      opt[String]('o', "output")
+        .action((v, c) =>
+          c.copy(outputStream = () => new BufferedOutputStream(new FileOutputStream(v))))
+        .text("output file")
+
+      opt[String]('m', "module")
+        .action((v, c) => c.copy(moduleDefs = v :: c.moduleDefs))
+        .text("module definition file or directory")
+
+      opt[String]('b', "base")
+        .action((v, c) => c.copy(baseDeclsFileName = v))
+        .text("path to the base declarations file")
+
+      opt[String]('D', "define")
+        .action((v, c) => c.copy(defines = c.defines + v))
+        .text("add a symbol to the conditional defines")
+
+      arg[String]("<file>")
+        .action((v, c) => c.copy(fileName = v))
+        .text("input file")
     }
 
+
     // Parse the options
     optParser.parse(args, Config()) map { config =>
```

Comment:
Since the `getOrElse` is gone, it is now more idiomatic (and efficient) to use `foreach` here instead of `map`.